BostonData/Intents/get_alerts.py

```diff
@@ -0,0 +1,32 @@
+from bs4 import BeautifulSoup
+import urllib.request
+
+"""
+
+City services in alerts dict:
```

Comment:
Is this file being used any more? It looks like the logic was moved into the `get_alert_intent.py`